### PR TITLE
J22 9 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/angelic_cub.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Angelic Cub
+ManaCost:1 W
+Types:Creature Cat Angel
+PT:1/1
+T:Mode$ BecomesTarget | ValidTarget$ Card.Self | TriggerZones$ Battlefield | FirstTime$ True | Execute$ TrigCounter | TriggerDescription$ Whenever CARDNAME becomes the target of a spell or ability for the first time each turn, put a +1/+1 counter on it.
+SVar:TrigCounter:DB$ PutCounter| Defined$ Self | CounterType$ P1P1 | CounterNum$ 1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/angelic_cub.txt:6]: spacing

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Types:Legendary Creature Devil Advisor
+PT:2/2
+K:Flying
+T:Mode$ DamageDoneOnce | Execute$ TrigCounter | ValidTarget$ You | ValidSource$ Card.YouCtrl,Emblem.YouCtrl | TriggerZones$ Battlefield | TriggerDescription$ Whenever a source you control deals damage to you, put that many +1/+1 counters on CARDNAME. 
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterAmount$ Y | CounterType$ P1P1
+SVar:Y:TriggerCount$DamageAmount
+A:AB$ DealDamage | Cost$ X T SubCounter<X/P1P1> | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to any target.
+SVar:X:Count$xPaid
+DeckHas:Ability$Counters
+Oracle:Flying\nWhenever a source you control deals damage to you, put that many +1/+1 counters on Auntie Blyte, Bad Influence. {1}{R}, {T}, Remove X +1/+1 counters from Auntie Blyte:It deals X damage to any target.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:12]: card name is spelled different than line 1

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Auntie Blythe, Bad Influence
+ManaCost:2 R
+Types:Legendary Creature Devil Advisor
+PT:2/2
+K:Flying
+T:Mode$ DamageDoneOnce | Execute$ TrigCounter | ValidTarget$ You | ValidSource$ Card.YouCtrl,Emblem.YouCtrl | TriggerZones$ Battlefield | TriggerDescription$ Whenever a source you control deals damage to you, put that many +1/+1 counters on CARDNAME. 
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterAmount$ Y | CounterType$ P1P1
+SVar:Y:TriggerCount$DamageAmount
+A:AB$ DealDamage | Cost$ X T SubCounter<X/P1P1> | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ X | SpellDescription$ CARDNAME deals X damage to any target.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:9]: `NICKNAME` in cost
`It` in SpellDesc

File: forge-gui/res/cardsfolder/upcoming/war_instrument.txt [hunk 4]
```diff
@@ -0,0 +1,9 @@
+Name:War Instrument
+ManaCost:4
+Types:Artifact
+K:Flash
+K:ETBReplacement:Other:ChooseCT
+SVar:ChooseCT:DB$ ChooseType | Defined$ You | Type$ Creature | AILogic$ MostProminentInComputerDeck | SpellDescription$ As CARDNAME enters the battlefield, choose a creature type.
+S:Mode$ Continuous | Affected$ Creature.ChosenType+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Creatures you control of the chosen type get +1/+1.
+SVar:PlayMain1:TRUE
+Oracle:Flash\nAs Instruments of War enters the battlefield, choose a creature type.\nCreatures you control of the chosen type get +1/+1.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/war_instrument.txt:9]: oracle name is different than filename and line 1

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt (hunk 3 above)
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:9]: Is this supposed to be Cost$ 1 R T SubCounter...
Doesn't match Oracle...

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt [hunk 5]
```diff
@@ -0,0 +1,12 @@
+Name:Auntie Blythe, Bad Influence
+ManaCost:2 R
+Types:Legendary Creature Devil Advisor
+PT:2/2
+K:Flying
+T:Mode$ DamageDoneOnce | Execute$ TrigCounter | ValidTarget$ You | ValidSource$ Card.YouCtrl,Emblem.YouCtrl | TriggerZones$ Battlefield | TriggerDescription$ Whenever a source you control deals damage to you, put that many +1/+1 counters on CARDNAME. 
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterAmount$ Y | CounterType$ P1P1
+SVar:Y:TriggerCount$DamageAmount
+A:AB$ DealDamage | Cost$ 1 R T SubCounter<X/P1P1> | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ X | SpellDescription$ NICKNAME deals X damage to any target.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:9]: SpellDescription should be `It deals...`
Cost probably `<X/P1P1/NICKNAME>

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+SVar:X:Count$xPaid
+DeckHas:Ability$Counters
+Oracle:Flying\nWhenever a source you control deals damage to you, put that many +1/+1 counters on Auntie Blythe, Bad Influence. {1}{R}, {T}, Remove X +1/+1 counters from Auntie Blyte: It deals X damage to any target.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:12]: Blyte still spelled wrong

File: forge-gui/res/cardsfolder/upcoming/instruments_of_war.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:War Instrument
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/instruments_of_war.txt:1]: still wrong

File: forge-gui/res/cardsfolder/upcoming/merfolk_pupil.txt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Name:Merfolk Pupil
+ManaCost:1 U
+Types:Creature Merfolk Wizard
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME enters the battlefield, draw a card, then discard a card.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose
+A:AB$ Draw | Cost$ 1 U  ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | SubAbility$ DBDiscard | SpellDescription$ Draw a card, then discard a card.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/merfolk_pupil.txt:8]: spacing

File: forge-gui/res/cardsfolder/upcoming/biblioplex_kraken.txt [hunk 9]
```diff
@@ -0,0 +1,13 @@
+Name:Biblioplex Kraken
+ManaCost:4 U
+Types:Creature Kraken
+PT:4/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigScry | TriggerDescription$ When CARDNAME enters the battlefield, scry 3.
+SVar:TrigScry:DB$ Scry | ScryNum$ 3
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ Whenever CARDNAME attacks, you may return another creature you control to it's owner's hand. If you do, CARDNAME can't be blocked this turn.
+SVar:TrigChangeZone:DB$ ChangeZone | Optional$ True | Origin$ Battlefield | Destination$ Hand | Hidden$ True | ChangeType$ Creature.YouCtrl+Other | SelectPrompt$ Select another creature you control | ChangeNum$ 1 | RememberChanged$ True | SubAbility$ DBUnblockable
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/biblioplex_kraken.txt:8]: make this into a Cost$ a la Slab Hammer

File: forge-gui/res/cardsfolder/upcoming/rampaging_growth.txt [hunk 10]
```diff
@@ -0,0 +1,7 @@
+Name:Rampaging Growth
+ManaCost:3 G
+Types:Instant
+A:SP$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Basic | RememberChanged$ True | ChangeNum$ 1 | SubAbility$ DBAnimate | SpellDescription$ Search your library for a basic land card and put that card onto the battlefield, then shuffle. Until end of turn, that land becomes a 4/3 Insect creature with reach and haste. It's still a land. 
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Power$ 4 | Toughness$ 3 | Types$ Creature,Insect | Colors$ Green | Keywords$ Haste & Reach | StackDescription$ None | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True 
+Oracle:Search your library for a basic land card and put that card onto the battlefield, then shuffle. Until end of turn, that land becomes a 4/3 Insect creature with reach and haste. It's still a land.
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rampaging_growth.txt:7]: DeckHas:Type$Insect

File: forge-gui/res/cardsfolder/upcoming/rampaging_growth.txt [hunk 11]
```diff
@@ -0,0 +1,7 @@
+Name:Rampaging Growth
+ManaCost:3 G
+Types:Instant
+A:SP$ ChangeZone | Origin$ Library | Destination$ Battlefield | ChangeType$ Land.Basic | RememberChanged$ True | ChangeNum$ 1 | SubAbility$ DBAnimate | SpellDescription$ Search your library for a basic land card and put that card onto the battlefield, then shuffle. Until end of turn, that land becomes a 4/3 Insect creature with reach and haste. It's still a land. 
+SVar:DBAnimate:DB$ Animate | Defined$ Remembered | Power$ 4 | Toughness$ 3 | Types$ Creature,Insect | Colors$ Green | Keywords$ Haste & Reach | StackDescription$ None | SubAbility$ DBCleanup
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/rampaging_growth.txt:5]: remove Color

File: forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt [hunk 12]
```diff
@@ -0,0 +1,12 @@
...
+ManaCost:2 R
+Types:Legendary Creature Devil Advisor
+PT:2/2
+K:Flying
+T:Mode$ DamageDoneOnce | Execute$ TrigCounter | ValidTarget$ You | ValidSource$ Card.YouCtrl,Emblem.YouCtrl | TriggerZones$ Battlefield | TriggerDescription$ Whenever a source you control deals damage to you, put that many +1/+1 counters on CARDNAME. 
+SVar:TrigCounter:DB$ PutCounter | Defined$ Self | CounterAmount$ Y | CounterType$ P1P1
+SVar:Y:TriggerCount$DamageAmount
+A:AB$ DealDamage | Cost$ 1 R T SubCounter<X/P1P1> | ValidTgts$ Creature,Player,Planeswalker | TgtPrompt$ Select any target | NumDmg$ X | SpellDescription$ NICKNAME deals X damage to any target.
+SVar:X:Count$xPaid
+DeckHas:Ability$Counters
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/auntie_blythe_bad_influence.txt:11]: probably AI:RemoveDeck:Random

File: forge-gui/res/cardsfolder/upcoming/biblioplex_kraken.txt [hunk 13]
```diff
@@ -0,0 +1,12 @@
+Name:Biblioplex Kraken
+ManaCost:4 U
+Types:Creature Kraken
+PT:4/5
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigScry | TriggerDescription$ When CARDNAME enters the battlefield, scry 3.
+SVar:TrigScry:DB$ Scry | ScryNum$ 3
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigChangeZone | TriggerDescription$ Whenever CARDNAME attacks, you may return another creature you control to it's owner's hand. If you do, CARDNAME can't be blocked this turn.
+SVar:TrigUnblockable:DB$ Effect | Cost$ Return<1/Creature> | RememberObjects$ TriggeredAttacker | ExileOnMoved$ Battlefield | StaticAbilities$ Unblockable | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/biblioplex_kraken.txt:8]: Cleanup no longer needed

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/biblioplex_kraken.txt:8]: `.Other`

File: forge-gui/res/cardsfolder/upcoming/merfolk_pupil.txt [hunk 14]
```diff
@@ -0,0 +1,12 @@
+Name:Merfolk Pupil
+ManaCost:1 U
+Types:Creature Merfolk Wizard
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ Whenever CARDNAME enters the battlefield, draw a card, then discard a card.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/merfolk_pupil.txt:5]: Whenever = When

**Review by tool4ever — APPROVED** [review]

**Pull request merged**